uvls/src/smt.rs

```diff
@@ -258,13 +258,13 @@ async fn find_fixed(
         "{}",
         smt_module_constraint.variable_to_source(&base_module)
     );
-    info!("{source_variable}");
+    //info!("{source_variable}");
```

Comment:
Is there any reason to not remove these lines of code instead of commenting them out?

Reply:
No has no reason, will be removed.